addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -20,6 +20,19 @@ import { usePopover } from "@web/core/popover/popover_hook";
 import { memoize } from "@web/core/utils/functions";
 import { escapeRegExp } from "@web/core/utils/strings";
 
+export function useEmojiPickerStoreScroll() {
+    const storeScroll = {
+        scrollValue: 0,
+        set: (value) => {
+            storeScroll.scrollValue = value;
+        },
+        get: () => {
+            return storeScroll.scrollValue;
+        },
+    };
+    return storeScroll;
+}
```

Comment:
This function has no reason to exists. It should be internal code of the `Emoji Picker`, as I said in previous comment